styles.css

```diff
@@ -113,33 +114,39 @@ textarea {
 
 input {
   padding: 10px;
+  border-radius: 4px;
```

Comment:
Make sure you're holding onto your convention here. If you're ordering your CSS properties alphabetically, this should go above padding.